src/test/java/uk/co/ramp/event/AlertEventProcessorTest.java

```diff
@@ -119,4 +132,32 @@ public void testClashingAlertStatus() {
     verify(population, never()).setAlertStatus(0, ALERTED);
     verifyNoMoreInteractions(population);
   }
+
+  @Test
+  public void determineTestResult() {
+
+    eventProcessor =
+        new AlertEventProcessor(
+            population, properties, diseaseProperties, distributionSampler, statisticsRecorder);
+
+    when(distributionSampler.uniformBetweenZeroAndOne()).thenReturn(0.99d);
+    assertThat(eventProcessor.determineTestResult(true))
+        .isNotEmpty()
+        .isEqualTo(Optional.of(TESTED_NEGATIVE));
```

Comment:
Think this reads better:
```suggestion
    assertThat(eventProcessor.determineTestResult(true)).hasValue(TESTED_NEGATIVE);
```

Could you change the other assertions below as well. Thanks!